Description: Increase the timeout to 10 minutes
Author: Stephen Kitt <skitt@debian.org>

This allows the package to build on mipsel buildds.

--- a/upstream/gas/testsuite/lib/gas-defs.exp
+++ b/upstream/gas/testsuite/lib/gas-defs.exp
@@ -330,7 +330,7 @@
 
 # Default timeout is 10 seconds, loses on a slow machine.  But some
 # configurations of dejagnu may override it.
-if {$timeout<120} then { set timeout 120 }
+if {$timeout<600} then { set timeout 600 }
 
 expect_after -i {
     timeout			{ perror "timeout" }
